<launch>
    <node pkg="dji_osdk_ros" type="dji_vehicle_node" name="vehicle_node" output="screen">
    <!-- node parameters -->
    <param name="acm_name" type="string" value="/dev/ttyACM0"/>
    <param name="serial_name" type="string" value="/dev/ttyTHS1"/>
    <param name="baud_rate" type="int" value="1000000"/>
    <param name="app_id" type="int" value="1106264"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="dd694ebc9089de602aa34505f6cde6644163425abe550b54ce10acd1dfd26ef9"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>
</launch>
